From e9b6cb871f7fbb8fc54f4fb63fe610d97270b51d Mon Sep 17 00:00:00 2001
From: zhouliang1 <zhouliang1@eswincomputing.com>
Date: Thu, 19 Jun 2025 19:10:40 +0800
Subject: [PATCH 569/652] feat:revert lpcpu modifications

Changelogs:
1.revert lpcpu modifications in linux dts and
buildroot of vpu7702 evb

Signed-off-by: zhouliang1 <zhouliang1@eswincomputing.com>
Reviewed-by: Joey Zheng <zhengyu@eswincomputing.com>
---
 arch/riscv/boot/dts/eswin/vpu7702-evb.dts | 8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/arch/riscv/boot/dts/eswin/vpu7702-evb.dts b/arch/riscv/boot/dts/eswin/vpu7702-evb.dts
index 21d2c8c083b67..a331af3f0898e 100644
--- a/arch/riscv/boot/dts/eswin/vpu7702-evb.dts
+++ b/arch/riscv/boot/dts/eswin/vpu7702-evb.dts
@@ -843,7 +843,7 @@ &fan_control {
 
 &d0_i2c0 {
 	/* codec es8388 */
-	status = "disabled";
+	status = "okay";
 	d0_es8388_0: es8388-0@10 {
 		compatible = "eswin,es8388";
 		reg = <0x10>;
@@ -892,7 +892,7 @@ npu_vcc1:npu_svcc{
 
 &d0_i2c2 {
 	/* touch screen*/
-	status = "disabled";
+	status = "okay";
 	gt911:touchscreen@14 {
 		compatible = "goodix,gt911";
 		reg = <0x14>;
@@ -907,7 +907,7 @@ gt911:touchscreen@14 {
 
 &d0_i2c3 {
 	/* mipi csi0/csi1 */
-	status = "disabled";
+	status = "okay";
 };
 
 &d0_i2c4 {
@@ -963,7 +963,7 @@ &d0_i2c9 {
 
 &d0_aon_i2c0 {
 	/* temp sensor & rtc */
-	status = "disabled";
+	status = "okay";
 	eswin,syscfg = <&d0_sys_con 0x3C0 16>;
 	pcf_rtc:rtc@51 {
 		compatible = "nxp,pcf8563";
-- 
2.53.0

